gobcore/datastore/postgres.py

```diff
@@ -1,3 +1,5 @@
+from typing import Iterator, Any
```

Comment:
`from typing import Iterator` is een deprecated alias to [collections.abc.Iterator](https://docs.python.org/3/library/collections.abc.html#collections.abc.Iterator).

Reply:
O didn't know this ;)
fixed